BSB_lan.ino

```diff
@@ -7795,8 +7796,9 @@ uint8_t pps_offset = 0;
     }
   }
   if(MQTTClient && mqtt_mode == 0){
-    delete MQTTClient;
     MQTTClient = NULL;
+    delete MQTTClient;
+    delete mqtt_client;
```

Comment:
Same as above
```
                        delete MQTTClient;
                        MQTTClient = NULL;
                        delete mqtt_client;
```

Reply:
Just to be sure: `mqtt_client->stop();` is still necessary, right?

Reply:
Yes, i mentioned about this in other comment.